pyro/logger.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import absolute_import, division, print_function
+
+import logging
+
+
+default_format = '%(levelname)s \t %(message)s'
+log = logging.getLogger("pyro")
+log.setLevel(logging.INFO)
```

Comment:
should this be configurable? at least between debug and info

Reply:
We can configure it as `pyro.log.setLevel(logging.DEBUG)`. I'm keeping `INFO` as default as a lot of diagnostic messages are passed in `INFO`.